### PR TITLE
add module logistic_requisition_department

### PR DESCRIPTION
**Pull request by lepistone**
dependencies:
- [x] OCA/department#27

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: logistic_requisition_department/model/logistic_requisition.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from openerp import models, fields, api
+
+
+class LogisticRequisition(models.Model):
+    _inherit = 'logistic.requisition'
+
+    def _get_my_department(self):
+        my_user = self.env['res.users'].browse(self.env.uid)
```
Comment (nbessi) [inline logistic_requisition_department/model/logistic_requisition.py]: self.env.user will be better

File: logistic_requisition_department/tests/test_propagate_department.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class TestPropagateDepartment(TransactionCase):
+    def setUp(self):
+        super(TestPropagateDepartment, self).setUp()
+
+        self.lrs = self.env['logistic.requisition.source'].new({
+            'name': '/',
+        })
+        self.dep_rd = self.browse_ref('hr.dep_rd')
```
Comment (nbessi) [inline logistic_requisition_department/tests/test_propagate_department.py]: I'll prefer  `self.env.ref('hr.dep_rd')` matter of taste non blocking

File: logistic_requisition_department/model/logistic_requisition.py (hunk 1 above)
Comment (lepistone) [inline logistic_requisition_department/model/logistic_requisition.py]: fixed

**Comment by nbessi** [comment]
:+1: I'll merge once green

**Comment by lepistone** [comment]
@nbessi I applied your suggestions, I agree (I didn't know about the `user` and `ref` properties of `Environment`). Thanks

**Pull request merged**